apps/vc-api/src/vc-api/credentials/credentials.service.spec.ts

```diff
@@ -187,39 +167,39 @@ describe('CredentialsService', () => {
       }
     });
     jest.spyOn(keyService, 'getPrivateKeyFromKeyId').mockResolvedValueOnce(key);
-    const vp = await service.provePresentation({ presentation, options: issuanceOptions });
-    expect(vc['proof']['jws']).toBeDefined();
+    const vp = await service.provePresentation({
+      presentation: rebeamPresentation,
+      options: issuanceOptions
+    });
+    expect(energyContractVerifiableCredential['proof']['jws']).toBeDefined();
     /**
      * Delete jws from proof as it is not deterministic
      * TODO: confirm this from the Ed25519Signature2018 spec
      */
     delete vp.proof.jws;
-    const expectedVpCopy = JSON.parse(JSON.stringify(expectedVp));
+    const expectedVpCopy = JSON.parse(JSON.stringify(vp));
     delete expectedVpCopy.proof.jws;
     expect(vp).toEqual(expectedVpCopy);
```

Comment:
`expectedVpCopy` will always be `vp` because you copied at line 180. What is the purpose of this test?

Reply:
They are supposed to be different. vp is created in test, expectedVp is pre-calculated. These names accidentally coincided during renaming. I fixed that